client/component/PlotComponent.tsx

```diff
@@ -144,6 +156,10 @@ class PlotComponent extends React.Component<PlotComponentProps, PlotStates> {
           console.log('data type is new image data');
           const newImageMessage = decoded_message as ImageDataMessage;
           this.plot_new_image_data(newImageMessage);
+      } else if ('ta_data' in decoded_message) {
+          console.log('data type is new matrix data');
```

Comment:
Good spot. Will correct and resolve conflict  when I get access to a bigger device. Don't try non-trivial ones with the web editor on a phone because it's unusable!

Reply:
L301, L303 and L305 too.